### ISSUE TITLE
Skipping sublabel indexing

### ISSUE THREAD

**Issue by LecrisUT** [holoviz/holoviews#6355]
#### Is your feature request related to a problem? Please describe.
Let's say I have a grid of 1x3 plots that use. Now I want to add some custom legend like a colorbar spanning the full layout for which I would construct a layout of layout, but alas I couldn't find how to do that. In this limited case I can just add it as the first overlay, but then it gets the additional sublabel. Ideally there would be a mechanism to skip the labeling of a specific overlay that is used in `+=`

#### Describe the solution you'd like
Have a `.opts()` option like `sublabel_label` or `sublabel_skip` that would overwrite the generation through `sublabel_format`.

#### Describe alternatives you've considered
Maybe this would be possible by composing with the primitives behind `holoview.Layout`, but how to use those is undocumented. The closest alternative is the `panel.Column`/`panel.Row`, but it is unclear how things like `renedere.save()` interact with those.

#### Additional context

![Screenshot_20240802_172543-1](https://github.com/user-attachments/assets/dfbe7a7a-3128-4622-8507-1b03ad0548ec)

**Comment by ahuang11** [comment]
Related to https://github.com/holoviz/hvplot/issues/1370

**Comment by LecrisUT** [comment]
Yes, but in this case it's about making it selectively skip labels, i.e. `b)` should become `a)` in the screenshot. Strongly related to [this discourse comment](https://discourse.holoviz.org/t/layout-of-layout/7531/9) though
